omnichannel-server/src/feature/message-source/MessageParser.ts

```diff
@@ -3,10 +3,18 @@ import { MessageTwilioPayload } from "./interfaces/MessageTwilioPayload";
 import { MessagePayload } from "./interfaces/MessagePayload";
 import { MessageProvider } from "./interfaces/MessageProvider";
 
-export const MessageParser = ({ body }) => {
+export class MessageParser {
+
+messageSource(req: Request, res: Response):Promise<Response>{
+  const payload = this.messageParser(req)
```

Comment:
Ya estás dentro de `MessageParser` entonces tu método podría llamarse sólo

```suggestion
  const payload = this.parse(req.body)
```

También 

```suggestion
  const payload = this.parseRequestBody(req.body)
```

```suggestion
  const payload = this.getMessagePayload(req.body);
```